### PR TITLE
Fix/buttons

### PR DESCRIPTION
**Pull request by ADavidson02**
## What is the change?
* disable the add button in the NewDream component after click
* Remove DreamCards from the dashboard and add the lineChart
* Remove the data page

## What does it fix?
* Adding the same dream multiple times

## Is this a fix or a feature? 
* Both 

## Where should the reviewer start?
Dashboard.js line 43

## How should this be tested?
npm test and check all tests are passing

### CODE REVIEW

File: src/modules/DreamJournal/DreamJournal.js [hunk 1]
```diff
@@ -19,39 +20,55 @@ const useStyles = makeStyles((theme) => ({
...
-      duration: theme.transitions.duration.shortest,
-    }),
+  // expand: {
+  //   transform: 'rotate(180deg)',
+  //   marginLeft: 'auto',
+  //   transition: theme.transitions.create('transform', {
+  //     duration: theme.transitions.duration.shortest,
+  //   }),
+  // },
+  loading: {
```
Comment (Shawntru) [inline src/modules/DreamJournal/DreamJournal.js:23]: Can we remove these comments?

File: src/modules/Header/Header.js [hunk 2]
```diff
@@ -1,16 +1,19 @@
...
-import Button from '@material-ui/core/Button';
+import {
+  AppBar,
+  Toolbar,
+  Typography,
+  IconButton,
+  Button,
+  MenuItem,
+  Menu,
+  useMediaQuery,
```
Comment (Shawntru) [inline src/modules/Header/Header.js:13]: Nice cleanup, thank you!

File: src/modules/NewDream/NewDream.js [hunk 3]
```diff
@@ -102,8 +118,9 @@ const NewDream = (props) => {
             }}
           ></TextField>
           {/* {error && <h6>{error}</h6>} */}
-          <Button variant="contained" color="primary" onClick={submitDream}>
+          <Button variant="contained" color="primary" disabled={disabled} onClick={submitDream}>
             Add
```
Comment (Shawntru) [inline src/modules/NewDream/NewDream.js]: possible to change to 
`{!loading && 'Add'}` ?

**Review by Shawntru — COMMENTED** [review]
Everything looks great! Let me know if possible to change that 'Add' text on button

File: src/modules/NewDream/NewDream.js (hunk 3 above)
Comment (ADavidson02) [reply src/modules/NewDream/NewDream.js]: Altered

File: src/modules/DreamJournal/DreamJournal.js (hunk 1 above)
Comment (ADavidson02) [reply src/modules/DreamJournal/DreamJournal.js:23]: Removed

**Pull request merged**